tests/conftest.py

```diff
@@ -918,3 +975,346 @@ def snakemake_fastqc_rule(tumor_only_config, helpers):
             )
         }
     )
+
+
+@pytest.fixture(scope="session", name="timestamp_now")
+def fixture_timestamp_now() -> datetime:
+    """Return a time stamp of today's date in date time format."""
+    return datetime.now()
+
+
+@pytest.fixture(scope="session", name="cosmic_key")
+def fixture_cosmic_key() -> str:
+    """Mocked COSMIC key."""
+    return "ZW1haWxAZXhhbXBsZS5jb206bXljb3NtaWNwYXNzd29yZAo="
+
+
+@pytest.fixture(scope="session", name="cluster_account")
+def fixture_cluster_account() -> str:
+    """Mocked cluster account for job submission."""
+    return "development"
+
+
+@pytest.fixture(scope="session", name="develop_containers")
+def fixture_develop_containers() -> Dict[str, str]:
+    """Develop containers fixture."""
```

Comment:
Or replace "dictionary" with something more meaningful, like mapping?
```suggestion
    """Return a Develop containers dictionary."""
```